src/rpc_client/src/meta_client.rs

```diff
@@ -112,7 +112,6 @@ impl MetaClient {
         self.set_worker_id(worker_node.id);
         // unpin snapshot before MAX will create a new snapshot with last committed epoch and then
         //  we do not create snapshot during every pin_snapshot.
```

Comment:
Does this comment still make sense?

Reply:
> Does this comment still make sense?

the comment means that we are replacing `pin_version` calls after `register` with `unpin_version_before`